pulpcore/cli/common/generic.py

```diff
@@ -151,3 +151,42 @@ def version_group(
 version_group.add_command(show_version)
 version_group.add_command(destroy_version)
 version_group.add_command(repair_version)
+
+
+# Generic label command group
+@click.group(name="label")
+@pass_entity_context
+@pass_pulp_context
+@click.pass_context
+def label_group(ctx: click.Context, entity_ctx: PulpEntityContext, pulp_ctx: PulpContext) -> None:
+    """Manage labels for this resource"""
+    pass
+
+
+@click.command(name="set")
```

Comment:
```suggestion
@label_group.command(name="set")
```
I think in the same file we can spare us the extra `add_command` (and the trouble of forgetting it).